src/shared/components/dateRangePicker/NewDatePicker.tsx

```diff
@@ -1,4 +1,5 @@
 import React, {FC, useState, useContext, useEffect, useCallback} from 'react'
+const moment = require('moment/min/moment.min.js')
```

Comment:
Is this what you mean with "strip down the bundle size" by using `require` instead of `import`?

Reply:
It's the pathing to the minimal js file, to be loaded. Otherwise we get moment bundled with all the different locales.

There are several different approaches in how to reduce a moment bundle size. This is a very easy one.